doc/source/whatsnew/v0.21.0.txt

```diff
@@ -332,6 +332,9 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Compatibility with PyPy increased by removing the assumption that sets are sorted in ``usecols``
```

Comment:
why don;t you make a separate bug fix sub-section for pypy fixes (IIRC a couple of more in here)

Reply:
you also don't need to be as detailed, give these from a user perspective, rather just refer to the issue (or PR in this case)